SeaPkg/Library/BasePeCoffValidationLib/BasePeCoffValidationLib.c

```diff
@@ -0,0 +1,451 @@
+/** @file
+  Defines function definitions for validating a loaded PE/COFF image post execution
+  against an auxiliary file (defined in SeaAuxiliary.h) that contains auxiliary

```

Comment:
I think this is supposed to say `contains multiple auxiliary validation entries`

Reply:
Yes, thanks. Fixed:)